lib/secretfile.rb

```diff
@@ -65,6 +65,7 @@ def read_spec
     raise "Expected Secretfile" unless File.readable?(spec_path)
     @spec = IO.readlines(spec_path).inject({}) do |memo, line|
       line.chomp!
+      next memo if line == ""
```

Comment:
`line =~ /\A\s*\z/`

Reply:
Doesn't `.chomp!` ensure whitespace is deleted?